File: frontend/src/app/models/create-customer-request.ts
export class CreateCustomerRequest {
  constructor(
    private username: string,
    private password: string,
    private firstName: string,
    private lastName: string,
    private telephone: string,
    private address: string,
    private city: string,
    private state: string,
    private zip: number,
    private creditCardNo: string,
    private email: string
  ) {
  }
}

File: backend/src/main/java/flightbook/entity/customer/CreateCustomerRequest.java
package flightbook.entity.customer;

import lombok.Data;

@Data
public class CreateCustomerRequest {
	private String username;
	private String password;

	private String firstName;
	private String lastName;
	private String telephone;
	private String address;
	private String city;
	private String state;
	private int zip;

	private String creditCardNo;
	private String email;
}
